tests-integration/reference-model/elm.json

```diff
@@ -4,6 +4,10 @@
         "../../src",
         "src"
     ],
+    "exposed-modules": [
+        "Morphir.Reference.Model.Types",
+        "Morphir.Reference.Model.Formulas"
+    ],
```

Comment:
Any specific reason for this?

Reply:
They need to be exposed in order to be used in a new project.

Reply:
I see. These are the modules you are referring to from your test project. Makes sense.